galaxytrucker.css

```diff
@@ -677,11 +681,13 @@ html {
 .alien_choice.chosen {
   width: 19px;
 }
+.content.crew.purple,
 .alien.purple,
 .notouch-device .alien_choice.purple.available:hover,
 .alien_choice.purple.chosen {
   background-position: -25px 0px;
 }
+.content.crew.brown,
```

Comment:
This css is such a mess. Any ideas on how to clean it up? Question 1: why do we need `.alien` and `.content.crew.purple`? Those should be the same thing, no?

Reply:
I have not looked more deeply into this, but asked myself the same question. For me the hierarchical crew -> human or crew -> alien -> purple/blue does not make too much sense since there is almost no concept in GT (except for crew placement) that needs to differentiate between a human and alien without taking color into account. So the DB model crew -> human/purple/blue makes more sense to me. I will merge this now, and I think we should keep the focus on getting the game running. Afterwards, we can clean up code.